### PR TITLE
Add Hotjar support

### PR DESCRIPTION
**Pull request by pjdelport**
This adds straightforward support for the Hotjar tracking code.

### CODE REVIEW

**Comment by pjdelport** [comment]
@bittner, @dufrenb: This should address #134.

File: analytical/templatetags/hotjar.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import re
+
+from django.template import Library, Node, TemplateSyntaxError
+
+from analytical.utils import get_required_setting, is_internal_ip, disable_html
+
+
+HOTJAR_TRACKING_CODE = """\
+<script>
+    (function(h,o,t,j,a,r){
```
Comment (bittner) [inline analytical/templatetags/hotjar.py]: Wow! That's funny! :smiley:

File: analytical/tests/test_tag_hotjar.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+from analytical.templatetags.analytical import _load_template_nodes
+from analytical.templatetags.hotjar import HotjarNode
+from analytical.tests.utils import TagTestCase
+from analytical.utils import AnalyticalException
+
+
+expected_html = """\
+<script>
+    (function(h,o,t,j,a,r){
```
Comment (bittner) [inline analytical/tests/test_tag_hotjar.py]: People from Malta have crazy funny ideas! :smirk:

**Comment by pjdelport** [comment]
Note that this is failing because of an unrelated new flake8 warning.

I'll make a PR for that too.

**Comment by bittner** [comment]
`flake8` [complains about](https://travis-ci.org/jcassee/django-analytical/jobs/455496596#L483) some [escape sequences](https://github.com/jcassee/django-analytical/blob/master/analytical/tests/test_utils.py#L33) in the test_utils. They should probably be raw strings (`r"\d+"`), not sure though. - Could you fix that, too? That would be helpful!

**Comment by pjdelport** [comment]
@bittner: Addressed in #138.

**Comment by bittner** [comment]
You may want to rebase the PR to make Travis happy. :+1:

**Pull request merged**